### PR TITLE
feat: Table Section and Breadcrum Unit test

### PR DESCRIPTION
**Pull request by jeheskielSunloy77**
1. Adding Table component on home page
2. Table data sorting
3. Breadcrumb and create new training unit test

Table section
Screenshot from 2022-08-16 19-35-00
Unit testing result
Screenshot from 2022-08-16 19-33-34

### CODE REVIEW

File: src/pages/Home/TableSection.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+				color: white;
+			}
+			.ant-table-thead > tr > th:hover {
+				background-color: #0780ee !important;
+			}
+		`}</style>
+	</>
+)
+
+const RatingScore = ({ score }) => {
```
Comment (argorfc) [inline src/pages/Home/TableSection.jsx:86]: Please movie into component folder, it should be a reusable component.

File: src/pages/Home/TableSection.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	const stars = [...Array(5)].map((_, index) => (
+		<StarIcon key={index} active={index < score} />
+	))
+
+	return stars
+}
+
+const StarIcon = ({ active }) => (
```
Comment (argorfc) [inline src/pages/Home/TableSection.jsx:94]: Please movie into component folder, it should be a reusable component.

File: src/pages/Home/TableSection.jsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+	<svg
```
Comment (argorfc) [inline src/pages/Home/TableSection.jsx:95]: create component icon, it can help for another icon

**Review by argorfc — CHANGES_REQUESTED** [review]
Please read comments!

**Comment by jeheskielSunloy77** [comment]
@argorfc i have made some changes and i have move TableData, RatingScore and StarIcons to their own component file.
thank you for the review

**Review by argorfc — COMMENTED** [review]
What is difference between src/components/StarIcon.jsx with  src/components/icons/StarIcon.jsx ?

**Comment by jeheskielSunloy77** [comment]
@argorfc Sorry for my mistake, I have deleted the file and the redundancy has been resolved.

**Review by argorfc — APPROVED** [review]
Approved.
Ready to merge.

**Pull request merged**